src/GafferArnold/IECoreArnoldPreview/ShaderNetworkAlgo.cpp

```diff
@@ -354,6 +356,14 @@ std::vector<AtNode *> convert( const IECoreScene::ShaderNetwork *shaderNetwork,
 			return AiNode( nodeType, nodeName, parentNode );
 		};
 		convertWalk( shaderNetwork->getOutput(), shaderNetwork, name, nodeCreator, result, converted );
+		if( const IECoreScene::Shader *surface = shaderNetwork->outputShader() )
+		{
+			// \todo: Should we generalize this to pass all blindData? Or perhaps just those with a particular prefix?
```

Comment:
I think we probably should. This is in the Cortex layer rather than the Gaffer one, so hardcoding Gaffer stuff is a little off.

Reply:
Sounds good, will do. Did you like that I'm prepending `user:` or should I just translate the keys verbatim?

Reply:
Addressed in 99c8d27ab84186119f2a5fc9019ec787401bcabb (I also dropped the `user:` prefix for now, but can add it back if you prefer.